sdk/include/opentelemetry/sdk/metrics/aggregation/default_aggregation.h

```diff
@@ -42,9 +42,8 @@ class DefaultAggregation
       case InstrumentType::kHistogram: {
         return (instrument_descriptor.value_type_ == InstrumentValueType::kLong)
                    ? std::move(std::unique_ptr<Aggregation>(new LongHistogramAggregation(
-                         static_cast<
-                             const opentelemetry::sdk::metrics::HistogramAggregationConfig<long> *>(
-                             aggregation_config))))
+                         static_cast<const opentelemetry::sdk::metrics::HistogramAggregationConfig<
+                             int64_t> *>(aggregation_config))))
```

Comment:
Nit - this need to be double after #1665 fix. Just adding, though it would be fixed once conflict is resolved.